### DISCUSSION TITLE
大佬，希望能不能给整理转移加个开关

### DISCUSSION THREAD

**Discussion by Heroxf** [jxxghp/MoviePilot#1190] [建议]
大佬，我个人不需要整理转移这个功能，能不能加个开关呀。现在我没配置媒体库目录和转移方式，总是会报错提示很烦，望大佬考虑一下
![tianruo_2023-12-1-638370497547483277](https://github.com/jxxghp/MoviePilot/assets/152601462/da0ce570-c729-45d5-a0dd-abf2705e0d39)

**Comment by Heroxf** [comment]
补充一下，我的nas是极空间，它自带的极影视可以直接刮削pt下载的资源，而且不改变文件内容，所以不需要转移整理功能

**Comment by cng10001** [comment]
你开了这个吗
![image](https://github.com/jxxghp/MoviePilot/assets/115156545/c781c19c-0e0a-4248-a23e-d5538afbfd61)

**Reply by Heroxf** [reply]
对，开了

**Reply by cng10001** [reply]
关了试试

**Reply by Heroxf** [reply]
好
